libs/node-sdk/package.json

```diff
@@ -23,7 +23,7 @@
     "dependencies": {
         "@featureboard/js-sdk": "workspace:0.10.0-beta.7",
         "debug": "^4.3.4",
-        "node-fetch": "^2.6.7",
+        "node-fetch": "^3.3.1",
```

Comment:
I think we should actually remove this (node-fetch) as a dependency.

fetch is now shipped in node 18, and we should just expect it to be polyfilled in other/older environments.